### PR TITLE
{bio}[intel/2021b] GTDB-Tk v2.0.0, HMMER v3.3.2, FastANI v1.33, FastTree v2.1.11 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node2615.swalot.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/bdfcf59cd5d4bae4c8c4ed26de8968a5 for a full test report.

File: easybuild/easyconfigs/g/GTDB-Tk/GTDB-Tk-2.0.0-intel-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    ('Python', '3.9.6'),
+    ('DendroPy', '4.5.2'),
+    ('matplotlib', '3.4.3'),
+    ('prodigal', '2.6.3'),
+    ('HMMER', '3.3.2'),
+    ('pplacer', '1.1.alpha19', '', True),
+    ('FastANI', '1.33'),
+    ('FastTree', '2.1.11'),
+    ('tqdm', '4.62.3'),
+]
```
Comment (branfosj) [inline easybuild/easyconfigs/g/GTDB-Tk/GTDB-Tk-2.0.0-intel-2021b.eb:29]: Based on https://ecogenomics.github.io/GTDBTk/installing/index.html#third-party-software - add [Mash](https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/m/Mash) as a dependency.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3508.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/c785a8d019af1bc6d3768919a9c8d46a for a full test report.

File: easybuild/easyconfigs/g/GTDB-Tk/GTDB-Tk-2.0.0-intel-2021b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/g/GTDB-Tk/GTDB-Tk-2.0.0-intel-2021b.eb:29]: Oh, nice catch...
Maybe we should also run `gtdbtk check_install` as a sanity check command to catch this, but it currently fails with:

```
The 'GTDBTK_DATA_PATH' environment variable is not defined.
```

Comment (branfosj) [reply easybuild/easyconfigs/g/GTDB-Tk/GTDB-Tk-2.0.0-intel-2021b.eb:29]: I spotted while doing  #15315 earlier today.

I tried a few options for a sanity check command, but all of them failed out - either on that env variable not being set or on the data not being as expected if I pointed it at a directory.

Comment (boegel) [reply easybuild/easyconfigs/g/GTDB-Tk/GTDB-Tk-2.0.0-intel-2021b.eb:30]: fixed in 997f868

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node2615.swalot.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/4fbb189677faf7503da6e9fa5905a133 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 8 out of 8 (5 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/4b085c7cf939bff4c662c5498c57fb55 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**